src/components/templates/OrderTemplate.jsx

```diff
@@ -0,0 +1,106 @@
+import { useMutation } from "react-query";
+import { comma } from "../../utils/convert";
+import { order } from "../../apis/order";
+import OrderList from "../molecules/OrderList";
+import { useState } from "react";
+import * as Order from '../../styles/templates/OrderTemplate';
+
+const OrderTemplate = ({data}) => {
+    const {products, totalPrice} = data?.data?.response;
+    const [agreePayment, setAgreePayment] = useState(false);
+    const [agreePolicy, setAgreePolicy] = useState(false);
+
+    const handleAllAgree = (e) => {
+        const checked = e.target.checked;
+
+        setAgreePayment(checked);
+        setAgreePolicy(checked);
+    }
+
+    const handleAgreement = (e) => {
+        const {name, checked} = e.target;
+
+        if(name === 'payment-agree') {
+            setAgreePayment(checked);
+        } else if (name === 'policy-agree') {
+            setAgreePolicy(checked);
+        }
+    }
+
+    const {mutate} = useMutation({
+        mutationFn: order,
+    });
```

Comment:
```suggestion
    const {mutate: mutateSomething} = useMutation({
        mutationFn: order,
    });
```
요케이름 바꿔주면 어떤 뮤테이션인지 알기 쉽습니다!
여러개의 뮤테이트가 있는경우도 마찬가지구요.